### PR TITLE
Autofac v5 Upgrade

### PR DESCRIPTION
**Pull request by alistairjevans**
Raise targeted framework version, appveyor update, nuspec constraint update.

### CODE REVIEW

File: src/Autofac.Integration.Mvc.Owin/Autofac.Integration.Mvc.Owin.nuspec [hunk 1]
```diff
@@ -14,8 +14,8 @@
     <iconUrl>https://cloud.githubusercontent.com/assets/1156571/13684110/16b8f152-e6bf-11e5-84ae-22c66c6d351a.png</iconUrl>
     <releaseNotes>Release notes are at https://github.com/autofac/Autofac.Mvc.Owin/releases</releaseNotes>
     <dependencies>
-      <dependency id="Autofac" version="[3.5.0,5.0.0)" />
-      <dependency id="Autofac.Owin" version="[3.1.0,5.0.0)" />
+      <dependency id="Autofac" version="[5.0.0,6.0.0)" />
+      <dependency id="Autofac.Owin" version="[5.0.0,6.0.0)" />
```
Comment (tillig) [inline src/Autofac.Integration.Mvc.Owin/Autofac.Integration.Mvc.Owin.nuspec]:
```suggestion
      <dependency id="Autofac.Owin" version="[5.0.1,6.0.0)" />
```

**Review by tillig — CHANGES_REQUESTED** [review]
Minor change to the nuspec to make sure folks don't get in a weird position and we're good.

File: src/Autofac.Integration.Mvc.Owin/Autofac.Integration.Mvc.Owin.nuspec (hunk 1 above)
Comment (alistairjevans) [reply src/Autofac.Integration.Mvc.Owin/Autofac.Integration.Mvc.Owin.nuspec]: Good shout; corrected.

**Review by tillig — APPROVED** [review]
🥇

**Pull request merged**